script/generate-authors

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env node
```

Comment:
I wanted to avoid nodejs, but turns out "get me the first paragraph of a markdown file" was a bit too much. If only there was like an `mdq` for markdown